.github/workflows/build-push-images.yaml

```diff
@@ -4,7 +4,13 @@ on:
   push:
     branches:
       - master
-      - release-*
+      - release-[12]*
```

Comment:
Why 2?

Reply:
have it, in case we'll move to release-2.Y in the future, according to @tiraboschi .
on the other hand, we can add it when relevant.